lib/getDigest.test.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+const { test } = require('tap')
+const getDigest = require('./getDigest')
+
+test('Constructs a matching Digest header from a valid request Digest header and body', async ({
+  equal
+}) => {
+  const tests = [
+    {
+      description: 'Matching Digest header constructed from an incoming request - sha256',
+      input: {
+        headers: {
+          digest: 'sha-256=k6I5cakU5erL8KjSUVTNownDwccvu5kU1Hxg88toFYg=',
+          'content-type': 'application/json'
```

Comment:
[nit] I think we can pull this content-type out.

Reply:
Yup. No longer necessary with the new assumptions for `getDigest`